src/ExtranetBundle/objects/TCMSTableEditor/TableEditorExtranetUser.php

```diff
@@ -210,4 +215,11 @@ private function router(): PortalAndLanguageAwareRouterInterface
     {
         return \ChameleonSystem\CoreBundle\ServiceLocator::get('chameleon_system_core.router.chameleon_frontend');
     }
+
+    private function isKernelSecretDefault(): bool
+    {
+        $secret = \ChameleonSystem\CoreBundle\ServiceLocator::getParameter('kernel.secret');
+
+        return '!ThisTokenIsNotSoSecretChangeIt!' === $secret;
```

Comment:
Maybe we should define this string in a constant somewhere. I do not mind the duplicated logic but the duplicated string should be avoided.

Reply:
:heavy_check_mark: The service now has a `isReadyToEncodeTokens` methods which is used in the talbe editor in order to prevent duplicate logic